extend/registration/index.md

```diff
@@ -127,16 +178,100 @@ This flag provides a UI for setting the file output mapping. With this UI, you c
+- Application Icon must be representative and reasonable quality. Make sure the icon license allows you to use it.
+- The application must correctly state the data flow --- UI flags `appInfo.dataOut` (typically writers), `appInfo.dataIn` (typically extractors).
+- Licensing information must be valid and vendor description must be current.
+
+
+#### Application Configuration
+
+- Use only the necessary flags (i.e. if there are no output files, don't use `genericDockerUI-fileOutput`).
+- For extractors, always use [Default bucket](/extend/common-interface/folders/#default-bucket) --- that means, don't use `genericDockerUI-tableOutput` flag.
+- Use [encryption](/overview/encryption/) to store sensitive values. No plain-text passwords!
+- Use [Configuration Schema](/extend/registration/configuration-schema/).
+    - List all properties in the `required` field.
+    - Always use `propertyOrder` to explicitly define the order of the fields in the form.
+    - Use short `title` without a colon.
+    - Use `description` to provide an explanatory sentence if needed.
+    <br>Good: ![Good Schema](/extend/registration/schema-good.png)
+    <br>Bad: ![Bad Schema](/extend/registration/schema-bad.png)
+- Use Configuration description only if the configuration is not trivial / self-explainable. Provide links to resources (e.g. when doing Elastic extractor, not everyone is familiar with ElasticSearch query syntax). The configuration description supports markdown. The markdown should not start with a header and should use only level 3 and level 4 headers (a level 2 header is prepended before the configuration description). <br>Good: <br><code>some introduction text<br><br>### Input Description<br>
+description of input tables<br>
+<br>
+#### First Table<br>
+some other text<br>
+</code>Bad:<br><code>
+## Configuration Description<br>
+some introduction text<br>
+<br>
+#### Input Description<br>
+description of input tables
+</code>
+
+
+#### Application Internals
+
+- Make sure that the amount consumed **memory does not depend** on the amount of processed data. Use streaming or processing in chunks to maintain a limited amount of consumed memory. If not possible, state the expected usage in the **Application Limits**.
+- The application must distinguish between [User and Application errors](/extend/common-interface/environment/#return-values).
+- The application must validate its parameters, invalid configuration must result in an user error.
+- The events produced must be reasonable. Provide status messages if possible and with reasonable frequency. Avoid internal messages with no meaning to the end-user. Avoid flooding the event log or sending data files in the event log.
+- Set up [Continuos Deployment](/extend/registration/deployment/) so that you can keep the application up-date.
+- Use [semantic versioning](http://semver.org/) to mark and deploy versions of your application. Do not use `latest` image tag in production.

```

Comment:
napsal bych, že použití obecných tagů jako `latest` nebo `master` je pro approval zakázaný